src/modules/class-wpml-cornerstone-accordion.php

```diff
@@ -2,13 +2,6 @@
 
 class WPML_Cornerstone_Accordion extends WPML_Cornerstone_Module_With_Items {
 
-	/**
-	 * @return string
-	 */
-	public function get_items_field() {
-		return '_modules';
```

Comment:
I noticed that we always have the same "items field" in Cornersone (different from other PB), and we can confirm this because this string `_modules` is also hardcoded in the main logic: https://github.com/OnTheGoSystems/wpml-page-builders-cornerstone/pull/19/files#diff-9287486d43abbdcc52b99a85786af1abL81/.

So I removed this abstract method from the parent class and we'll now use a class constant.